packages/client-sdk-nodejs/src/internal/cache-data-client.ts

```diff
@@ -785,8 +786,9 @@ export class CacheDataClient implements IDataClient {
     cacheName: string,
     key: string | Uint8Array,
     value: string | Uint8Array,
-    ttl?: number
+    options?: SetIfAbsentCallOptions
```

Comment:
Breaking change? Just flagging make sure ok.. I know this is newer api we just added.

Reply:
Nope. This is the internal data client API. The public API is in `[ICacheClient](https://github.com/momentohq/client-sdk-javascript/blob/626da3c65b09b2b1e90a9864a996bc026ec73c70/packages/core/src/internal/clients/cache/AbstractCacheClient.ts#L730-L753)` which is unchanged. Good eye though 👍